modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/index.js

```diff
@@ -224,10 +224,14 @@ export function parseOptions(jsonString) {
 	}
 
 	return Array.isArray(options)
-		? options.map(({value}) => `${value}`).join(', ')
+		? options.filter((option) => !!option.value.length)
 		: options;
 }
 
+export function parseValue(value) {
+	return Array.isArray(value) ? value.join(', ') : value;
```

Comment:
Probably is better to filter the array to avoid null/empty items otherwise you will have unwanted commas

`
[1,null,undefined,,0,'',2].join(", ") ---> '1, , , , 0, , 2'
[1,null,undefined,,0,'',2].filter(item => item === 0 || item).join(", ") ---> '1, 0, 2'
`